src/ffi/abstract_.rs

```diff
@@ -3,6 +3,12 @@ use crate::ffi::pyport::Py_ssize_t;
 use std::os::raw::{c_char, c_int};
 use std::ptr;
 
+#[cfg_attr(windows, link(name = "pythonXY"))]
+extern "C" {
+    #[cfg_attr(PyPy, link_name = "PyPySequence_Type")]
+    pub static mut PySequence_Type: PyTypeObject;
```

Comment:
There is no `PySequence_Type` in the CPython API.  This is because "sequence" is not a concrete type, it's an abstract API that works with any type that has the corresponding `tp_as_sequence` members implemented.

Reply:
Definitely that was my understanding and what I did here is due to tiredness... I'm going to come up with an alternative idea.  Thanks for pointing it out.

Reply:
I think I managed to fix this.